kolena/_utils/state.py

```diff
@@ -129,21 +119,17 @@ def get_client_state() -> _ClientState:
     return CLIENT_STATE.get(_client_state)
 
 
-def is_client_initialized() -> bool:
-    try:
-        get_client_state().assert_initialized()
-    except (InvalidClientStateError, UninitializedError):
-        return False
-    return True
+def is_client_uninitialized() -> bool:
+    return get_client_state().jwt_token is None
```

Comment:
Should this not follow the same logic in the removed `assert_initialized`?

Reply:
I view the `base_url` guard as dead code, we provide a default in the constructor (probably shouldn't be an optional) https://github.com/kolenaIO/kolena/blob/trunk/kolena/_utils/state.py#L54.
What really signals the client is initialized is the presence of a jwt token, the auth token is a required to retrieve the jwt token https://github.com/kolenaIO/kolena/blob/trunk/kolena/_utils/state.py#L186